### PR TITLE
fix wechat videoplayer pause&stop&play bug

### PR DESCRIPTION
**Pull request by novain1102**
Re: # https://github.com/cocos/3d-tasks/issues/11994

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js [hunk 1]
```diff
@@ -291,7 +291,9 @@ if (cc.internal.VideoPlayer) {
             let self = this;
             let video = this._video;
             if (!video || !this._visible) return;
-
+            if (!this._playing) { // to unify performance of android and ios
+                video.play();
+            }
```
Comment (minggo) [inline platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js:298]: Sorry, I don't quite understand the modification. What does `to unify performance of android and ios` mean?

Comment (novain1102) [reply platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js:298]: on Android, video won't play from begin if called pause() before called stop(). 
But if called play() before stop(), video will play from begin.
I guess this inconsistency is caused by wechat internal api desgin.
detailed description: https://github.com/cocos/3d-tasks/issues/11994#issuecomment-1628347125

Comment (minggo) [reply platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js:298]: Got it, so i have two suggestions:
- add more detail information for the comment
- the comment seems wrong, to unify behavior, not performance, right?

Comment (novain1102) [reply platforms/minigame/platforms/wechat/wrapper/engine/VideoPlayer.js:298]: > add more detail information for the comment

Got it.

> the comment seems wrong, to unify behavior, not performance, right?

Yes. You're right.

**Review by minggo — APPROVED** [review]

**Pull request merged**